### PR TITLE
InfluxMetricsProvider cache multiple sets of data

### PR DESCRIPTION
**Pull request by tagantroy**
Fix for #185. 
Thanks @memega 

1. Remove unused import
2. Fix code format

### CODE REVIEW

**Comment by Malinskiy** [comment]
@tagantroy can we add at least one test to check the expected behaviour which:
1. Cache if missing
2. Return from cache if present with the same params
3. Call again if missing in cache because of new params

File: core/src/main/kotlin/com/malinskiy/marathon/analytics/metrics/remote/RemoteDataSource.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.malinskiy.marathon.analytics.metrics.remote
+
+import java.time.Instant
+
+data class ExecutionTime(val testName: String, var percentile: Double)
```
Comment (Vacxe) [inline core/src/main/kotlin/com/malinskiy/marathon/analytics/metrics/remote/RemoteDataSource.kt]: Will be great to split it to different files

**Comment by tagantroy** [comment]
Hello @Malinskiy.
I added 2 tests per metric:

1. Return from cache if present with the same params
2. Call again if missing in cache because of new params

> Cache if missing 

Covered by "Return from cache if present with the same params"

**Review by Malinskiy — APPROVED** [review]

**Comment by Malinskiy** [comment]
Fixes #254

**Pull request merged**